setup.py

```diff
@@ -82,13 +83,9 @@ def write_version_py(filename=DEFAULT_VERSION_FILE,
         git_rev, dev_num = git_version()
     elif os.path.exists(DEFAULT_VERSION_FILE):
         # must be a source distribution, use existing version file
-        try:
-            from scimath._version import git_revision as git_rev
-            from scimath._version import full_version as full_v
-        except ImportError:
-            raise ImportError("Unable to import git_revision. Try removing "
```

Comment:
I don't think we need this any more: it may have been relevant when there were version files around that didn't include the git revision, but at this point it's safe to assume that if the version file exists in the expected location, then it was generated by this setup script and _does_ include the git revision.